pkgs/applications/misc/survex/default.nix

```diff
@@ -0,0 +1,59 @@
+{ lib
+, stdenv
+, autoreconfHook
+, pkg-config
+, wxGTK30-gtk3
+, ffmpeg
+, proj
+, perl532
+, unscii
+, python
+, libGL
+, libGLU
+, x11
+, docbook2x
+, docbook5
+}:
+
+stdenv.mkDerivation rec {
+  pname = "survex";
+  version = "1.2.44";
+
+  nativeBuildInputs = [ docbook5 x11 libGL libGLU docbook2x autoreconfHook pkg-config wxGTK30-gtk3 ffmpeg proj python (perl532.withPackages (perlPackages: with perlPackages; [ LocalePO ] )) ];
```

Comment:
Some inputs looks like that they should go into `buildInputs`.

Reply:
Probably most of them. I did not understand the distinction between `nativeBuildInputs` and `buildInputs`, now that I do I will correct this.

Reply:
How's this look? 42e27ff15da3dd376f19008c18a68ada0df854d3